pkg/image/validation/os.go

```diff
@@ -46,6 +46,12 @@ func validateKernelArgs(os *image.OperatingSystem) []FailedValidation {
 					UserMessage: "Kernel arguments must be specified as 'key=value'.",
 				})
 			}
+
+			if key == "fips" && !os.EnableFips {
```

Comment:
Should we also check the value? Isn't `fips=0` something that might be intentionally set?

Reply:
Hmm, good call.